.github/workflows/test_sitl_copter.yml

```diff
@@ -2,20 +2,135 @@ name: test copter
 
 on:
   push:
-    paths-ignore: # ignore vehicle only changes not for copter
+    paths-ignore:
+      # remove other vehicles
       - 'AntennaTracker/**'
       - 'ArduPlane/**'
       - 'ArduSub/**'
       - 'Rover/**'
       - 'Blimp/**'
+      # remove non SITL HAL
+      - 'libraries/AP_HAL_ChibiOS/**'
+      - 'libraries/AP_HAL_ESP32/**'
+      # remove non SITL directories
+      - 'Tools/AP_Bootloader/**'
+      - 'Tools/AP_Periph/**'
+      - 'Tools/CHDK-Script/**'
+      - 'Tools/CPUInfo/**'
+      - 'Tools/CodeStyle/**'
+      - 'Tools/FilterTestTool/**'
+      - 'Tools/Frame_params/**'
+      - 'Tools/GIT_Test/**'
+      - 'Tools/Hello/**'
+      - 'Tools/IO_Firmware/**'
+      - 'Tools/Linux_HAL_Essentials/**'
+      - 'Tools/LogAnalyzer/**'
+      - 'Tools/Pozyx/**'
+      - 'Tools/PrintVersion.py'
+      - 'Tools/Replay/**'
```

Comment:
This doesn't look right; we test Replay as part of our Copter SITL testing, don't we?

Reply:
Rigth, I wasn't aware, we got a Replay workflow  https://github.com/ArduPilot/ardupilot/blob/master/.github/workflows/test_replay.yml so I didn't think that replay was hidden into copter test. We should probably correct this into antoher PR